cli/cmd/sync.go

```diff
@@ -191,5 +192,21 @@ func syncConnection(ctx context.Context, sourceSpec specs.Source, destinationsSp
 	fmt.Printf("Summary: resources: %d, errors: %d, panic: %d, failed_writes: %d, time: %s\n", totalResources, summary.Errors, summary.Panics, failedWrites, tt.Truncate(time.Second).String())
 	log.Info().Str("source", sourceSpec.Name).Strs("destinations", sourceSpec.Destinations).
 		Int("resources", totalResources).Uint64("errors", summary.Errors).Uint64("panic", summary.Panics).Uint64("failed_writes", failedWrites).Float64("time_took", tt.Seconds()).Msg("Sync completed successfully")
+	if err := analyticsClient.Enqueue(analytics.Track{
```

Comment:
Shouldn't this be under `sync` and not `syncConnection`. Meaning we'd like to track each invocation of the `sync` command by every user, and not the invocation of each source?

Then we can send the list of sources and list of destinations in a single event

Reply:
I think we only have `sync` as there is no such thing as `sync_source` a sync is always source->to multiple dests

It's just a cli feature that can do those in a loop but for analytics purposes it's all the same event (there wont be a `sync` event and a `sync_source` event)